app/views/steps/client/benefit_check_result/edit.html.erb

```diff
@@ -1,12 +1,18 @@
 <% title t('.page_title') %>
 <% step_header %>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <p>Benefit check result: <%= @form_object.benefit_check_result[:benefit_checker_status] %></p>
+<div class="govuk-grid-row app-inverted--card app-inverted-text">
+  <div class="govuk-grid-column app-inverted-text">
+
     <%= govuk_error_summary(@form_object) %>
     <%= step_form @form_object do |f| %>
-      <%= f.continue_button %>
+      <% if @form_object.benefit_check_result[:benefit_checker_status] == 'Yes' %>
+        <%= render 'passported' %>
```

Comment:
I find the use of partials here a bit confusing. It hinders the readability IMO.

I presume it is so the different results (passported / non-passported) can have different titles, and one is coming from the `govuk_radio_buttons_fieldset` whereas the other (when no radio fieldset) is coming from an explicit `h1`?

I'm thinking perhaps it may be worth exploring, even if more verbose, branching out into 2 separate routes/controllers/formobjects, one for passported, another for non-passported?
And we decide which one to go to in the decision tree?

Reply:
I initially thought that the differences were going to be more substantial than they were. I can move the content from the partials onto the edit page and do away with them. I don't think the difference between the two is substantial enough to warrant two separate routes/controller/formobjects though.